tests/Neo.VM.Tests/UT_Utility.cs

```diff
@@ -54,7 +54,7 @@ public void TestGetBitLength()
             var random = new Random();
 
             // Big Number (net standard didn't work)
-            VerifyGetBitLength(BigInteger.One << 32 << int.MaxValue, 2147483680);
+            Assert.ThrowsException<OverflowException>(() => VerifyGetBitLength(BigInteger.One << 32 << int.MaxValue, 2147483680));
```

Comment:
why is this change? the behavor is no longer the same?

Reply:
`BigInteger.One << 32 << int.MaxValue` will throw OverflowException in .net 9

Reply:
Then this test is useless